### PR TITLE
added expansion warning

### PR DESCRIPTION
**Pull request by lennessyy**
Added expansion warning

### CODE REVIEW

**Review by shanesTG — CHANGES_REQUESTED** [review]
Should say "deleted a vertex or edge attribute"

File: modules/cluster-resizing/pages/expand-a-cluster.adoc [hunk 1]
```diff
@@ -17,6 +17,21 @@ WARNING: If your TigerGraph cluster uses a seeded license, please https://tigerg
 * Obtain a few key measures for the state of your data before the expansion, such as vertex counts/edge counts or certain query results. This will be useful in verifying data integrity after the expansion completes.
 * If the original cluster is a single node installation, make sure the IP used is not a https://en.wikipedia.org/wiki/Localhost[local loopback address] such as 127.0.0.1.
 
+[WARNING]
+====
+If you ever deleted a vertex attribute, cluster expansion might lead to data corruption.
```
Comment (a-m-thomas) [inline modules/cluster-resizing/pages/expand-a-cluster.adoc:22]: Maybe "if you have ever deleted a vertex attribute..." 
Do we need to say "This is a known issue"?

File: modules/cluster-resizing/pages/expand-a-cluster.adoc [hunk 2]
```diff
@@ -17,6 +17,21 @@ WARNING: If your TigerGraph cluster uses a seeded license, please https://tigerg
...
+Prior to expansion, run the following command as the TigerGraph Linux user on your instance to check if you can proceed safely.
+Make sure all TigerGraph services are running when you run the script:
```
Comment (a-m-thomas) [inline modules/cluster-resizing/pages/expand-a-cluster.adoc:24]: Maybe explain here how to make sure all services are running? "Use the `gadmin status` command to make sure..."

xref to https://docs.tigergraph.com/tigergraph-server/current/system-management/management-with-gadmin#_gadmin_status

File: modules/release-notes/pages/index.adoc [hunk 3]
```diff
@@ -131,6 +131,12 @@ See xref:gsql-ref:querying:syntax-versions.adoc[] for a summary of the differenc
 
 == Known Issues
 
+=== Cluster expansion
+
+If you ever deleted a vertex attribute before conducting an expansion, the expansion might lead to data corruption.
+
```
Comment (a-m-thomas) [inline modules/release-notes/pages/index.adoc:137]: "If you have ever deleted..."

**Review by alexthomasTG — APPROVED** [review]
Well explained!

**Review by ElliotMtg2011 — APPROVED** [review]
LGTM after changing data corruption wording.

**Pull request merged**